server/bin/immich-healthcheck

```diff
@@ -1,3 +1,23 @@
 #!/usr/bin/env bash
 
-node /usr/src/app/dist/bin/healthcheck.js
+if [[ "$IMMICH_WORKERS_EXCLUDE" == *"api"* ]] ||
+   [[ "$IMMICH_WORKERS_INCLUDE" != *"api"* ]]; then
+        echo "API worker excluded, skipping";
+        exit 0;
+fi
+
+IMMICH_HOST="${IMMICH_HOST:=localhost}"
```

Comment:
This should be `:-`, not `:=`. Same on next line.

https://tldp.org/LDP/abs/html/parameter-substitution.html